ui/lib/pki/addon/components/pki-role-form.hbs

```diff
@@ -8,7 +8,7 @@
         {{! DEFAULT VIEW }}
         {{#if (eq group "default")}}
           {{#each fields as |attr|}}
-            {{#if (eq attr.name "issuerRef")}}
+            {{#if (and (eq attr.name "issuerRef") @issuers.length)}}
```

Comment:
⚡ Just as an fyi empty arrays are considered falsy in hbs so we could just check for `@issuers` here
<img width="778" alt="Screenshot 2023-05-03 at 11 56 54 AM" src="https://user-images.githubusercontent.com/68122737/236016307-4c7a40e4-2299-4bc6-be67-a5a03bafc8bb.png">
 ([docs](https://guides.emberjs.com/release/components/conditional-content/))